src/System.Private.CoreLib/shared/System/Globalization/CultureData.Windows.cs

```diff
@@ -66,9 +66,9 @@ private unsafe bool InitCultureData()
 
             int result;
             string realNameBuffer = _sRealName;
-            char* pBuffer = stackalloc char[LOCALE_NAME_MAX_LENGTH];
+            char* pBuffer = stackalloc char[LocaleNameMaxLength];
```

Comment:
LOCALE_NAME_MAX_LENGTH is the exact name of a Windows const.  It should really be an one of the interop files, but for now at least it shouldn't have its casing changed.

Reply:
So unix also uses this but I've replaced the occurences of this in the Windows code with interop variables correctly named